backend/app/services/adapters/task_kinds/operations.py

```
@@ -1154,23 +1283,34 @@ def create_task_id(self, db: Session, user_id: int) -> int:
         Create new task id using tasks table auto increment.
         """
         try:
-            return task_store.create_placeholder_task_id(db, user_id=user_id)
+            return task_stores.task_store.create_placeholder_task_id(
+                db, user_id=user_id
+            )
 
+        except TaskIdAllocationError as e:
+            db.rollback()
+            raise HTTPException(
+                status_code=503, detail=f"Unable to allocate task ID: {str(e)}"
```

Ruff (0.15.17)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)